Taskfile.yml

```diff
@@ -300,12 +297,25 @@ tasks:
     desc: Build & serve the playground.
     dir: web/playground
     cmds:
-      # Must set `install-links=false` in the playground's `npm install` to
-      # install prql-js as the regular dependency, instead of creating a
-      # symlink. Refer to https://github.com/PRQL/prql/pull/1296.
-      - npm install --install-links=false
+      - task: install-playground-npm-dependencies
```

Comment:
I was thinking this wouldn't be the default under `run-playground` — there would be a separate task `run-playground-npm-dependencies` 

The default needs to pick up the full path, otherwise it would be v confusing that it's using old rust code. 

I'm not sure it worth investing much in us detecting the rust changes, since it might be difficult and it will be replaced by the proper solution when we can do https://github.com/PRQL/prql/issues/1836

Reply:
Hmmm... This PR is meant as a convenience for people doing documentation. The current `main` branch _always_ runs the full npm install (taking ~1 minute), for both `task run-web` and `task run-playground`, even if I ^C the one, start the other, with no changes to any of the source files.

On further inspection, I see there's a `task run-website` that does what I want. It simply serves out the (previously-built) static site. I have created a similar `task run-pg` that people can use to avoid the npm step completely.

I believe it's useful to keep the separate `install-playground-npm-dependencies` task because it factors out the (unexpected) `--install-links` option into a single function.

Potential To-do's:

1. Detect changes to `prql-compiler` source file in `install-playground-npm-dependencies`
2. Regularize the task names
   - `build-web` - builds the website (the current `build-web`)
   - `build-run-web` - builds and runs the website (the current `run-web`)
   - `run-web` - run previously-built static site (the current `run-website`)
   - similar names for build-playground, build-run-playground, run-playground
   - similar names for run-book, just for completeness

Reply:
(responding quickly, forgive me for brevity!)

> * `build-web` - builds the website (the current `build-web`)
> 
>     * `build-run-web` - builds and runs the website (the current `run-web`)
> 
>     * `run-web` - run previously-built static site (the current `run-website`)

I've been referring to `web` as all three — book / playground / website — and `website` as just the static site (at prql-lang.org). Does that change your view at all?

> Detect changes to `prql-compiler` source file in `install-playground-npm-dependencies`

As I say above — if you want to do this then you're welcome to, but it'll go away shortly when #1836 is unblocked (by Debian rust version...)



> On further inspection, I see there's a `task run-website` that does what I want. It simply serves out the (previously-built) static site.

The `run-` steps _also_ build things IIUC. But yes OK; we could have `run-` skip the building and have `build-run` do both? (Though, again, soon building will be very cheap, so it might be a smaller change to just add your new task as a suffix of `-cached` and then we refactor it out when we do #1836?)